source/replication/replication-overview.rst

```diff
@@ -37,6 +41,13 @@ server-side replication *requires* the source and destination bucket be MinIO
 clusters. The source and destination bucket *may* be the same MinIO cluster *or*
 two independent MinIO clusters.
 
+For each write operation to the bucket, MinIO checks all configured replication
```

Comment:
@poornas very minor addition for the metadata replication. A more complete migration of the design doc will be a separate PR.